externs/lcevc.js

```diff
@@ -0,0 +1,93 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+'use strict';
+
+var libDPIModule = {};
+var LcevcDil = {};
+
+/**
+* LCEVC DIL constructor
+* @constructor
+*/
+LcevcDil.LcevcDIL = class {
+  /**
+     *  @param {HTMLVideoElement} media
+     *  @param {HTMLCanvasElement} canvas
+     *  @param {Object=} dilConfig
+     */
+  constructor(media, canvas, dilConfig) {
+    /** @type {number} */
+    this.aspectRatio;
+
+    /** @type {Boolean} */
+    this.isFullscreen;
+
+    /** @type {Object} */
+    this.support;
+  }
+
+  /**
+   * Append the fMP4 fragments
+   * after they are appended to Media Source Extensions
+   * SourceBuffer. Here the lcevc data will be extracted and manged in line with
+   * the ranges given by the SourceBuffer.buffered() call result
+   *
+   * @param {!BufferSource} data fMP4 fragment.
+   * @param {string} type The type of the fragment.
+   * @param {!number} level The level of the fragment.
+   */
+  appendBuffer(data, type, level) {}
+
+  /**
+   * Set current variant as level to the LCEVC decoder
+   * @param {!number} level The variant of the fragment.
+   */
+  setCurrentLevel(level) {}
+
+  /**
+   * Set current variant as level to the LCEVC decoder
+   * @param {!number} level
+   * @param {!number} autoBufferSwitch
+   */
+  setLevelSwitching(level, autoBufferSwitch) {}
+
+  /**
+   * Set container Format for LCEVC Data.
+   * @param {!number} containerFormat if enhancement data travels in container
+   */
+  setContainerFormat(containerFormat) {}
+
+  /**
+   * Close the DIL
+   */
+  close() {}
+};
+
+/**
+* LCEVC Support Check
+*/
+LcevcDil.SupportObject = {
+
+  /**
+   * Check if canvas has WebGL support
+   *  @param {HTMLCanvasElement} canvas
+   */
+  webGLSupport(canvas) {},
+
+};
+
+/**
+* LCEVC Support Checklist Result
+* @type {Boolean}
+*/
+LcevcDil.SupportStatus;
+
+/**
+* LCEVC Support CheckList Error if any.
+* @type {String}
```

Comment:
`string` (lowercase) differs from `String` (uppercase) in the same way as I described `boolean` vs `Boolean` above

Reply:
This is rectified.